### DISCUSSION TITLE
cypress over docker documentation shows a very old image

### DISCUSSION THREAD

**Discussion by sivanbecker** [cypress-io/cypress#14427] [Docker] [answered]
Hi,

First i would like to thank you for the great framework. i'm new but enjoy it very much.
after using a local installation of cypress through npm i decided to try out the docker option and found that the documentation talks about image tag 3.2.0. ( https://www.cypress.io/blog/2019/05/02/run-cypress-with-a-single-docker-command)
At first i just used this and my test failed (cy.readFile failed to find an existing file) and after checking i saw many newer versions of docker images at docker hub. I tried 6.0.0 and my test passed. 

My question: is there a reason why the docs talk about this old version ?is that only because no one updated the docs or is it really you preferred/official "included"  version.

Thanks for your time
Sivan

**Comment by bahmutov** [comment, marked as answer]
Thank you,

So the blog posts are like snapshots in time, showing how to perform something. We do not go back and update every blog post whenever a new version comes out. The principles of that blog post still stand though, and recently I have updated the example repository linked from the blog post https://github.com/bahmutov/demo-docker-cypress-included so that's up to date.

**Reply by sivanbecker** [reply]
Oh, i got to the docker blog post through a link in the docs ,somehow i missed the fact it was a blog and not part of the docs.
Thanks for the quick answer :)
